Makefile

```diff
@@ -7,4 +7,8 @@ kind-create:
 	./scripts/init-kind-cluster.sh
 
 terraform-hack-init:
-	./hack/init.sh
\ No newline at end of file
+	./hack/init.sh
+
+integration-tests:
+	# TODO automatically run the operator
+	ginkgo --progress
```

Comment:
Do we need Ginkgo in the devcontainer?

Reply:
Oops - missed a commit. Will add that in the next PR!